Resources/Textures/_RMC14/Mobs/Felinids/felinid_ears.rsi/meta.json

```diff
@@ -0,0 +1,75 @@
+{
+    "version": 1,
+    "copyright": "@Vordenburg",
```

Comment:
Were these made by Vordenburg? I know SS13 had felinids, but if these specific sprites (see other meta.json's too) were made by Vordenburg then this is correct.

Reply:
as far as I can tell, the person in question is unable to be reached to ask them. However after looking and asking around, as far as I am able to find this is accurate yes.